.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "airbnb",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "semi": [2, "never"],
+    "no-param-reassign": 0
```

Comment:
You're right, I also don't like polluting code with eslint related comments but *maybe* in this case is better. This mutation is the official way to change comments with babylon (I'm not sure the `leadingComments` node is already initialized) and is the only point in the code we mutate a received parameter. If it's ok for you I'd re-enable the rule and I'll use an `eslint-disable-line` instruction. If you don't feel comfortable with this approach let the rule as it is now.